107/src/test/java/com/pany/domain/Client.java

```diff
@@ -23,6 +23,8 @@
  */
 public class Client implements Serializable {
 
+  private static final long serialVersionUID = 1L;
```

Comment:
Could we use the generated one, I think IntelliJ can do that.

Reply:
Yes. But since it's a test, there's no need don't you think?